### ISSUE TITLE
A simple simulation

### ISSUE THREAD

**Issue by bnorthan** [WeisongZhao/rl_positivity_sim#1]
Thanks for starting this repository.  This is a great idea.  Here is a simulation I think would be useful.  Create two points, distance d apart.  Apply a forward operator (convolution with PSF, sampling, noise, etc.), then apply the inverse model (RL).  Start with large d and decrease the distance d.    For given PSF, pixel size and noise what is the minimum distance d, for which you can still restore 2 points?

**Comment by WeisongZhao** [comment]
Good point, considering the two-point structure is hard to visualize, may be two-line is a  better choice. Please see our **Supplementary Fig. 2** and **6** in the [manuscript](https://www.researchsquare.com/article/rs-279271/v1).
![image](https://user-images.githubusercontent.com/38429821/110942361-aefa0d00-8374-11eb-93e5-41f9fc97e51a.png)
![image](https://user-images.githubusercontent.com/38429821/110942657-14e69480-8375-11eb-9762-2a6b550b19a9.png)

**Comment by bnorthan** [comment]
Yes.  The two line is a great example.  I just realized that your repository is a fork of https://github.com/jdmanton/rl_positivity_sim.

I forked it too a while ago and looked at Wiener Filter and SSIM (https://github.com/bnorthan/rl_positivity_sim/commits/master).  

It looks like the simulations are already very mature.  Since I worked with code before anyway, I can probably change it myself to look at different structure (like points instead of lines).  Looking forward to reading more of your analysis.
